modules/custom/az_paragraphs/az_paragraphs_text_media/js/az_paragraphs_az_text_media_vimeo.es6.js

```diff
@@ -0,0 +1,189 @@
+(($, Drupal) => {
+  Drupal.behaviors.az_vimeo_video_bg = {
+    attach(context, settings) {
+      if (window.screen && window.screen.width > 768) {
+        // @see https://developer.vimeo.com/player/sdk/basics
+        // Defaults
+        const defaults = {
+          ratio: 16 / 9,
+          vimeoId: '',
+          mute: true,
+          repeat: true,
+          width: $(window).width(),
+          playButtonClass: 'az-video-play',
+          pauseButtonClass: 'az-video-pause',
+          start: 0,
+          minimumSupportedWidth: 600,
```

Comment:
I recommend updating the defaults to add the `background` parameter and use `loop` instead of `repeat` (to match the parameters here: https://developer.vimeo.com/player/sdk/embed). I also reordered them a little and I think it would be good to use the same order when we initialize the player below: I'll add one more comment to update that.

```suggestion
          vimeoId: '',
          width: $(window).width(),
          ratio: 16 / 9,
          autoplay: true,
          background: true,
          loop: true,
          playButtonClass: 'az-video-play',
          pauseButtonClass: 'az-video-pause',
          minimumSupportedWidth: 600,
```